services/pdf-ms/helpers/caseDataConverter.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Checks if string is a numeric value
+ * @param {string} str
+ */
+const isNumeric = (str: string): boolean => {
+  if (typeof str !== 'string') return false;
+  // eslint-disable-next-line no-restricted-globals
+  return !isNaN(parseFloat(str));
+};
+
+/**
+ * Converts case answers array to object.
+ * @param {array} answerArray the flat array that we want to convert to a nested tree structure
+ */
+export const convertAnswerArrayToObject = (
+  answerArray: { field: { id: string; tags: string[] }; value: string }[]
+): Record<string, any> => {
+  const caseObject: Record<string, any> = {};
+  if (!Array.isArray(answerArray)) {
+    return caseObject;
+  }
```

Comment:
Plenty of noice words like answerArray, caseObject etc.
.....................................................^^^^^^..........^^^^^^^